terraform/modules/gro_ingestion_service/lambdas/src/models/EventRecord.ts

```diff
@@ -0,0 +1,45 @@
+import {AttributeValue} from "aws-lambda/trigger/dynamodb-stream"
+
+export interface EventRecord {
+    hash: string
+
+    RegistrationId: string
+    EventTime: string
+    DateOfDeath: string
+    VerificationLevel: string
+    PartialMonthOfDeath: string
+    PartialYearOfDeath: string
+
+    FirstForename: string
+    Surname: string
+    MaidenSurname: string
+    Sex: string
+    DateOfBirth: string
+    AddressLine1: string
+    AddressLine2: string
+    AddressLine3: string
+    AddressLine4: string
+    Postcode: string
+}
+
+export const mapToEventRecord = (image: { [key: string]: AttributeValue }): EventRecord => {
+    return {
+        hash: image["hash"].S ?? "",
```

Comment:
`image` feels like an odd name, is there a convention I've completely missed?

Reply:
Yep, this is dynamodbs naming, they call each entry an image, so on the stream event they have a `NewImage` and an `OldImage`